examples/vault-variable-test/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "vault-variable-test"
+authors = ["tpmccallum <tim.mccallum@fermyon.com>"]
```

Comment:
I'd suggest removing this from examples in the Spin repo (although I don't have strong feelings, just glancing at a couple of other random examples)